news/_posts/releases/2019-07-04-prestashop-1-7-6-0-rc2-release.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "Release note 1.7.6 RC2"
+subtitle: "This second release candidate for the 1.7.6 fixes the regressions identified in the RC1."
+date:   2019-07-04 16:30:00
+authors: [ PrestaShop ]
+icon: icon-leaf
+tags:
+ - version
+ - rc
+ - minor
+ - releases
+ - 1.7.6.x
+---
+
+Once again the PrestaShop community actively participated in the testing of the PrestaShop 1.7.6 Release Candidate.
+Thanks to your involvement, 3 regressions were identified and fixed.
+
+Our team is glad to announce PrestaShop 1.7.6.0 Release Candidate 2.
+If no new regression is reported by the end of the week (27), we will make the final version of 1.7.6.0 available next week (28).
```

Comment:
@samuel-pires Since it's Friday, do we keep this part ? The end of the week is Sunday ?

Reply:
I removed the "end of the week" part.
Thanks!